README.md

```diff
@@ -45,3 +45,129 @@ This repository contains helm charts for various deployment types of the tip wla
  - Run the following command under tip-wlan-helm directory _after_ the components are running:
  	- helm test <RELEASE_NAME> -n default
 	(For more details add --debug flag to the above command) 
+
+
+# Local environment
+
+In `wlan-pki-cert-scripts` repository edit the following files and add/replace strings as specified below:
+
+```
+mqtt-server.cnf:
+
+-commonName_default   = opensync-mqtt-broker.zone1.lab.wlan.tip.build
++commonName_default   = opensync-mqtt-broker.wlan.local
+
+
+openssl-server.cnf:
+-DNS.1  = opensync-redirector.zone1.lab.wlan.tip.build
+-DNS.2  = opensync-controller.zone1.lab.wlan.tip.build
++DNS.1  = opensync-redirector.wlan.local
++DNS.2  = opensync-controller.wlan.local
+ DNS.3  = tip-wlan-postgresql
+-DNS.4  = ftp.example.com
+```
+
+In `wlan-pki-cert-scripts` repository run `./generate_all.sh` to generate CA and certificates, then run `./copy-certs-to-helm.sh <local path to wlan-cloud-helm repo>` in order to copy certificates to helm charts.
+
+Optionally, in order to speedup first and subsequent runs, you may cache some images:
+
+```
+minikube cache add zookeeper:3.5.5
+minikube cache add bitnami/postgresql:11.8.0-debian-10-r58
+minikube cache add postgres:latest
+minikube cache add gcr.io/k8s-minikube/storage-provisioner:v3
+minikube cache add eclipse-mosquitto:latest
+minikube cache add opsfleet/depends-on
+```
+
+These images may occasionally need to be updated with these commands:
+
+```
+minikube cache reload ## reload images from the upstream
+eval $( minikube docker-env )
+for img in $( docker images --format '{{.Repository}}:{{.Tag}}' | egrep 'busybox|alpine|confluentinc/cp-kafka|zookeeper|k8s.gcr.io/pause|nginx/nginx-ingress|bitnami/cassandra|bitnami/postgresql|postgres|bitnami/minideb' ); do
+	minikube cache add $img;
+done
+```
+
+Run minikube:
+
+```minikube start --memory=10g --cpus=4 --driver=virtualbox --extra-config=kubelet.serialize-image-pulls=false --extra-config=kubelet.image-pull-progress-deadline=3m0s --docker-opt=max-concurrent-downloads=10```
+
+Deploy CloudSDK chart:
+
+```helm install tip-wlan tip-wlan -f tip-wlan/resources/environments/dev-local.yaml -n default```
+
+Wait a few minutes, when all pods are in `Running` state, obtain web ui link with `minikube service tip-wlan-wlan-cloud-static-portal -n tip --url`, open in the browser. Importing or trusting certificate might be needed.
+
+Services may be exposed to the local machine and local network with ssh, kubectl or kubefwd port forwarding, needs to be repeated for each service, please examples below:
+
+Kubefwd:
```

Comment:
I would mention what Kubefwd does and also note that this is for development purposes only and that in real use cases exposing the services "outside" Kubernetes infra will be done through the loadbalancer or with MetalLB